mxnet/training/docker/1.8.0/py3/Dockerfile.cpu

```diff
@@ -135,12 +135,12 @@ RUN ${PIP} install --no-cache --upgrade \
 
 # Install extra packages
 RUN pip install --no-cache-dir -U \
-    bokeh \
-    imageio \
-    opencv-python \
-    plotly \
-    seaborn \
-    shap
+   "bokeh>=2.3,<3" \
+   "imageio>=2.9,<3" \
+   "opencv-python>=4.3,<5" \
+   "plotly>=5.1,<6" \
+   "seaborn>=0.11,<1" \
+   "shap>=0.39,<1"
```

Comment:
nit: Please fix indentation by placing 4 spaces before the start of `"..." \` in each line here.

Reply:
will do, will wait for CI and then will add these changes.